examples/with-joi/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "with-joi",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "joi": "17.4.0",
+    "next": "10.2.3",
```

Comment:
Could you update all deps to latest?

Reply:
Thanks for the review, @leerob! I did the change.